javascript/node/selenium-webdriver/test/lib/capabilities_test.js

```diff
@@ -126,4 +127,21 @@ describe('Capabilities', function() {
       assert.deepEqual({bar: 123}, caps[Symbols.serialize]());
     });
   });
+
+  describe('Test StrictFileInteractability capability', function(env){
+    it('Test Uploading With Invisible File Input When StrictFileInteractability Is On', async function(){
+      let options = new chrome.Options;
+      options.setStrictFileInteractability(true);
+      var driver = env.builder().setChromeOptions(options).build();
+
+      driver.setFileDetector(new remote.FileDetector);
+      await driver.get(Pages.uploadInvisibleTestPage);
+      var input = await driver.findElement(By.id("upload"));
+      try{
+        await input.sendKeys(fp);
```

Comment:
This can silently fail, if the `sendKeys` won't throw the test will pass incorrectly.

Add `assert(false, "element was interactable")`

Reply:
Updated in both tests